src/python/nimbusml/pipeline.py

```diff
@@ -819,6 +819,12 @@ def _fit_graph(self, X, y, verbose, **params):
                     model=output_predictor_model)
                 output_predictor_model_node._implicit = True
                 graph_nodes.append(output_predictor_model_node)
+        elif learner_node == None and transform_models[-1].startswith("$dataset_transformer_model"):
+            single_model_node = transforms_modelcombiner(
```

Comment:
I dont understand why we need this change?

Reply:
We need this to deal with situations where there are no predictors directly in the Pipeline (excluding situations where the predictor is given to a DatasetTransformer in its model to transform) and a DatasetTransformer is the last item in the Pipeline, as shown in the test cases in test_datasettransformer_with_predictor_model.py .